app/views/devise/registrations/edit.html.slim

```diff
@@ -27,44 +27,45 @@
             .actions
               = f.submit('Update', class: 'btn btn-primary', disabled: true)
 
-    #add_application_token_form.collapse
-      = form_for :application_token, url: application_tokens_path, remote: true, html: {id: 'new-application-token-form', class: 'form-horizontal', role: 'form'} do |f|
-          .form-group
-            = f.label :application, {class: 'control-label col-md-2'}
-            .col-md-7
-              = f.text_field(:application, class: 'form-control', required: true, placeholder: "Name")
-          .form-group
-            .col-md-offset-2.col-md-7
-              = f.submit('Create', class: 'btn btn-primary')
+    .app-token-wrapper
```

Comment:
Instead of wrapping the form, why don't you simply add this class into the form itself ?